### DISCUSSION TITLE
Relationship between PyGObject and PyGObject-stubs versioning

### DISCUSSION THREAD

**Discussion by on2e** [pygobject/pygobject-stubs#200] [Q&A]
Hello,

What is the relationship between the versioning of the `PyGObject` package and its corresponding `PyGObject-stubs` package?

For instance, if a developer is working with `PyGObject==3.46.0`, which version of `PyGObject-stubs` should they install via `pip` to ensure compatibility?

Thank you!

**Comment by lovetox** [comment]
stubs target the latest stable released version for PyGObject.

This can be different for other libraries, for example we support both Gtk3 and Gtk4, you can check the README on how to learn on how to install the correct stubs.
